### DISCUSSION TITLE
changing the account name

### DISCUSSION THREAD

**Discussion by nickorzha** [community/community#62782] [Repositories]
### Select Topic Area

Question

### Body

Is it possible to rename the account name of Github?
How to change the account name?

**Comment by jankozik** [comment]
Yes, it is possible to change the account name (username) on GitHub. Here's how you can change your GitHub account name:

1. Sign in to your GitHub account: Go to the GitHub website (github.com) and sign in with your existing account credentials.

2. Access your account settings: Once you are signed in, click on your profile picture in the top-right corner of the page. From the dropdown menu, select "Settings."

3. Navigate to the "Account" settings: In the left sidebar of the settings page, click on the "Account" tab.

4. Change your username: On the Account settings page, you will see your current username under the "Username" section. Click on the "Change username" button next to it.

5. Choose a new username: Enter your desired new username in the provided field. Keep in mind that the new username must meet GitHub's username requirements, such as being unique and not violating any trademark or copyright.

6. Verify the new username: After entering the new username, GitHub will check its availability. If the username is available, a green checkmark will appear, indicating that it can be used. If the username is not available, you will need to choose a different one.

7. Confirm the changes: Once you have selected an available username, click on the "Change username" button to confirm the changes.

8. Verify your password: GitHub will prompt you to enter your account password to confirm the username change. Enter your password and click "Confirm password" to proceed.

9. Username change complete: After confirming your password, GitHub will process the username change. Once completed, your account will have the new username.

Please note that changing your GitHub account name will also update the URL of your GitHub profile and any repositories associated with your account. Additionally, any links or references to your previous username will no longer work.

It's important to consider the impact of changing your username, as it may affect your existing projects, collaborations, and links to your GitHub account.

**Reply by nickorzha** [reply]
It's helpfull to me. thanks.

**Comment by elderguardian** [comment]
Hello,

If you are logged in you can change your username at https://github.com/settings/admin.

**Reply by nickorzha** [reply]
thanks for your help.

**Comment by goldest-star** [comment]
I have another way.
If you are logged in you can change your username at https://github.com/settings/admin.

**Reply by nickorzha** [reply]
ok.

**Comment by queenofcorgis** [comment]
@nickorzha @jankozik GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
